resolver/resolver.cc

```diff
@@ -3963,7 +3963,7 @@ class CheckAbstractClassInitialization {
             auto method_new = singletonClass.data(ctx)->findMethodTransitive(ctx.state, core::Names::new_());
             // If the .new method we find is owned by Class, that means
             // there was no user defined .new method, which warrants an error.
-            if (method_new.data(ctx)->owner.show(ctx) == "Class") {
+            if (method_new.data(ctx)->owner.show(ctx) == core::Symbols::Class().toString(ctx)) {
```

Comment:
I strongly suspect that you want something like:
```cpp
if (method_new.data(ctx)->owner == core::Symbols::Class()) {
```
instead. There should be no need to do string comparisons when the symbols are unique.